### DISCUSSION TITLE
Are batched writes available?

### DISCUSSION THREAD

**Discussion by okletstryit** [capawesome-team/capacitor-firebase#582] [Q&A] [answered]
https://firebase.google.com/docs/firestore/manage-data/transactions#batched-writes
Can I use web API if I need this feature?

Thanks for your work!

**Comment by robingenz** [comment, marked as answer]
Batch writes are not yet supported by the plugin, see https://github.com/capawesome-team/capacitor-firebase/issues/444. 
You can use this plugin in combination with the Firebase JS SDK if you also authenticate your users on the web layer.
